src/main/java/bit/javaoop/FixedBudgetManager.java

```diff
@@ -0,0 +1,49 @@
+package bit.javaoop;
+
+public class FixedBudgetManager extends Manager {
+
+    private double fixedBudget;
+
+    public FixedBudgetManager(String name, double salary) {
+        super(name, salary);
+    }
+
+    public boolean isSatisfied() {
+        if (this.getSalary() > 20_000 && (this.currentBudget() < 15_000)) {
```

Comment:
Można uprościć tego ifa alt + enter na nim:D potem jeszcze raz alt + enter i add clarifying parentheses potem ekstrakcja liczb do stałych i spójrz jak ładnie to wygląda:
![image](https://user-images.githubusercontent.com/18233796/37984093-9899cf38-31f5-11e8-94de-5dabdb5f75b2.png)